src/utils/constants.ts

```diff
@@ -1,13 +1,10 @@
-const baseAPIURL: string = import.meta.env.NB_API_QUERY_URL;
-export const queryURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}query?`
-  : `${baseAPIURL}/query?`;
-export const attributesURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}attributes/`
-  : `${baseAPIURL}/attributes/`;
-export const nodesURL: string = baseAPIURL.endsWith('/')
-  ? `${baseAPIURL}nodes`
-  : `${baseAPIURL}/nodes`;
+export const baseAPIURL: string = import.meta.env.NB_API_QUERY_URL.endsWith('/')
+  ? import.meta.env.NB_API_QUERY_URL
+  : `${import.meta.env.NB_API_QUERY_URL}/`;
+
+export const queryURL: string = `${baseAPIURL}query?`;
```

Comment:
I don't think we did. Will check.

Reply:
It seems like this is the least verbose approach since other tools/libraries like `url-join`, `ulijs` are designed for more complex scenarios and are unnecessary for this purpose.